DependencyInjection/Configuration.php

```diff
@@ -796,6 +797,10 @@ public function getConfigTreeBuilder(): TreeBuilder
                 ->ifTrue(function ($v) { return 'server_log' === $v['type'] && empty($v['host']); })
                 ->thenInvalid('The host has to be specified to use a ServerLogHandler')
             ->end()
+            ->validate()
+                ->ifTrue(function ($v) { return 'console' !== $v['type'] && $v['interactive_only']; })
```

Comment:
@stof is this where we check for bridge version being >= 7.3, is there an example in the bundle already?

Reply:
@stof do you have any guidance here?

Reply:
either find a class or interface allowing to easily detect that with a `class_exists` or `interface_exists` check.

Otherwise, an alternative is to add a dependency on `composer-runtime-api` and use their `InstalledVersions` API